Content.Shared/Atmos/Rotting/ProRottingContainerComponent.cs

```diff
@@ -0,0 +1,12 @@
+namespace Content.Shared.Atmos.Rotting;
+
+/// <summary>
+/// Entities inside this container will rot at a faster pace, e.g. a grave
+/// </summary>
+[RegisterComponent]
+public sealed partial class ProRottingContainerComponent : Component
```

Comment:
Shared components should be networked please, also technically this comment is wrong as mentioned above because only the outermost container matters.